src/docfx/lib/log/Error.cs

```diff
@@ -42,15 +42,16 @@ internal class Error
             PullRequestOnly = pullRequestOnly;
         }
 
-        public Error WithCustomRule(CustomRule customRule, bool? isCanonicalVersion = null)
+        public Error WithCustomRule(CustomRule customRule, bool? enabled = null)
```

Comment:
>enabled [](start = 65, length = 7)

CustomRule has a field `Disabled`, which may have some conflicts with the `enabled` parameter?

Reply:
the field has been respected before convert rules to json schema.

https://github.com/dotnet/docfx/blob/v3/src/docfx/config/ops/OpsMetadataRuleConverter.cs#L58-L63